knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to view Web Session Summary
+description: An article on how to capture and inspect web traffic using Fiddler Everywhere
+type: how-to
+slug: how-to-capture-and-inspect-web-traffic
+tags: capture web traffic, inspect web traffic, web traffic, https, web application
+publish: true
+res_type: kb
+---
+
+## View Sessions Summary
+
+Each __Session__ captured by Fiddler Everywhere has multiple attributes ordered in columns. Use these to identify HTTP related issues, to track and debug your site performance, and to leave custom comments.
+
+## (ID)
+
+An auto-generated identification number for the currently listed sessions.
+
+## Result
+
+Indicates the HTTP status code from the response. A server issues the status codes in response to a client's request made to the server and represented by a three-digit number. The first digit of the status code specifies one of five standard classes of responses.
+
+- __1xx__ informational response – the request was received, continuing process
+- __2xx__ successful – the request was successfully received, understood, and accepted
+- __3xx__ redirection – further action needs to be taken to complete the request
+- __4xx__ client error – the request contains bad syntax or cannot be fulfilled
+- __5xx__ server error – the server failed to fulfill a valid request
+
+Full list of all response status code is available at the [HTTP/1.1 status code definition article](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html)
+
+## Protocol
+
+Indicates the used protocol, which can be __HTTP__ (Hypertext Transfer Protocol) or __HTTPS__ (secure HTTP). The __HTTPS__ is HTTP but with an added secure layer, which is usually TLS (or SSL). The secure layer encrypts the standard HTTP requests and responses, making them far more reliable than the plain-text HTTP. Fiddler can decore the encrypted content, but for this to happen, you must install a trust root certificate. Once the trust certificate is installed and the HTTPS capturing is enabled, Fiddler is effectively a "man-in-the-middle" and can now decode encrypted sessions by switching on the __Decode__ button.
```

Comment:
_decore_ > **decode** 

We usually avoid using words like **must** in the documentation,
```
Fiddler can decode the encrypted content, but a trust root certificate should be installed for this to happen
```